packages/display/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@tailor-cms/tced-vue",
```

Comment:
Is this name OK or is something else better?

Reply:
Consider renaming to something like tced-custom-element